libs/dyn/merge/merge_test.go

```diff
@@ -27,6 +29,15 @@ func TestMergeMaps(t *testing.T) {
 			"bar": "qux",
 			"qux": "foo",
 		}, out.AsAny())
+
+		// Locations of both values should be preserved.
+		assert.Equal(t, []dyn.Location{l2, l1}, out.Locations())
```

Comment:
This is now reversed, I presume. It was defined first in v1, so the location of v1 comes first.

Reply:
Yeah, exactly. Changed the test.